junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/LauncherDiscoveryRequestBuilder.java

```diff
@@ -198,6 +199,30 @@ public LauncherDiscoveryRequestBuilder configurationParameters(Map<String, Strin
 		return this;
 	}
 
+	/**
+	 * Add the supplied configuration parameters resource file to the request.
+	 * @param propertiesFile the classpath location of the properties file
+	 * never {@code null}
+	 * @return this builder for method chaining
+	 */
+	public LauncherDiscoveryRequestBuilder configurationParametersResource(String propertiesFile) {
+		Preconditions.notNull(propertiesFile, "properties file must not be null");
+		configurationParametersResources.add(propertiesFile);
+		return this;
+	}
+
+	/**
+	 * Add all of the supplied configuration parameters resource files to the request.
+	 * @param propertiesFiles the classpath locations of the properties files
+	 * never {@code null}
+	 * @return this builder for method chaining
+	 */
+	public LauncherDiscoveryRequestBuilder configurationParametersResources(List<String> propertiesFiles) {
```

Comment:
How about var-args instead of a list?

You could then get rid of the `configurationParametersResource(String propertiesFile)` variant.

Also, please rename `propertiesFiles` to `paths`.